### PR TITLE
LPS-154599 [Raylife D2C] Create Raylife Quote object

### PR DESCRIPTION
**Pull request by eliasantosf**
Story: [LPS-154599](https://issues.liferay.com/browse/LPS-154599)

### CODE REVIEW

**Comment by eliasantosf** [comment]
ci:test:sf

**Review by joanagrego — CHANGES_REQUESTED** [review]

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-definitions/quote.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+			"indexedLanguageId": "en_US",
+			"label": {
+				"en_US": "Billing Option"
+			},
+			"listTypeDefinitionId": "[$LIST_TYPE_DEFINITION_ID:BillingOption$]",
+			"name": "billingOption",
+			"required": false,
+			"type": "String"
+		},
+		{
```
[the remark this answers is not in the document]
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-definitions/quote.json:21]: Done!

File: modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-definitions/quote.json [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+			"indexedAsKeyword": false,
+			"indexedLanguageId": "",
+			"label": {
+				"en_US": "Data JSON"
+			},
+			"name": "dataJSON",
+			"required": false,
+			"type": "Clob"
+		},
+		{
```
[the remark this answers is not in the document]
Comment (eliasantosf) [reply modules/apps/site-initializer/site-initializer-raylife-d2c/src/main/resources/site-initializer/object-definitions/quote.json:34]: Done!

**Comment by eliasantosf** [comment]
ci:test:sf

**Review by joanagrego — APPROVED** [review]
Nice!

**Review by silasbsi — APPROVED** [review]
LGTM :fire:

**Comment by silasbsi** [comment]
ci:forward

**Pull request closed without merging**